README.md

```diff
@@ -48,6 +48,8 @@ NAME                      DESCRIPTION             VERSION(S)
 cluster.etcd.coreos.com   Managed etcd clusters   v1beta1
 ```
 
+Note that the etcd clusters managed by the etcd operator will not be deleted if the operator fails or the deployment of the operator is deleted. This is an intentional design to prevent accidental operator failure from killing all the etcd clusters.
```

Comment:
there is also the failure part which is different from the deleting deployment. Also i want to point it is by design not a flaw or something. we should do it since people asked about why before.